app/src/main/java/com/banuba/sdk/example/beautification/camera/CameraViewModel.kt

```diff
@@ -286,18 +286,22 @@ class CameraViewModel(application: Application) : AndroidViewModel(application)
         }
     }
 
-    private fun callJsMethod(feature: Feature) {
+    private fun evalJs(feature: Feature) {
         when (feature) {
-            is SimpleFeature -> callJsMethod(feature.method, feature.parameter)
-            is LinearFeature -> callJsMethod(feature.method, feature.value.toString())
-            is ColorFeature -> callJsMethod(feature.method, getColor(feature.color))
+            is SimpleFeature -> evalJs(feature.method, feature.parameter)
+            is LinearFeature -> evalJs(feature.method, feature.value.toString())
+            is ColorFeature -> evalJs(feature.method, getColor(feature.color))
         }
     }
 
-    private fun callJsMethod(method: String, parameter: String) {
-        effect?.callJsMethod(method, parameter)
+    private fun makeJsScript(method: String, params: String?): String {
```

Comment:
JS - Java Script
JsScript - Java Script Script ???
Rename please. For example "makeEvalJsCode"